packages/website/src/services/useAccountsUrl.js

```diff
@@ -3,6 +3,7 @@ import skynetClient from "./skynetClient";
 
 export default function useAccountsUrl() {
   const [url, setUrl] = React.useState("");
+  const createAccountsUrl = React.useCallback((path = "") => url && new URL(path, url).toString(), [url]);
```

Comment:
Can we unit test this?

Reply:
I see that there were a couple additional commits to fix bugs which is why we need automated tests for functions like this. It's easy to get logic wrong or miss edge cases.

Reply:
I skipped on unit testing components due to time constraints, maybe now that we have one more frontend dev on board we will be able to do better in this regard.